fixtures/stairville/xbrick-quad-16x8w-rgbw.json

```
@@ -0,0 +1,797 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "xBrick Quad 16x8W RGBW",
+  "shortName": "xBrick RGBW",
+  "categories": ["Color Changer", "Strobe"],
+  "meta": {
+    "authors": ["Maximilian B."],
+    "createDate": "2023-02-11",
+    "lastModifyDate": "2023-02-11"
+  },
+  "comment": "4 Clusters of 4 RGBW LEDs (4x4)",
+  "links": {
+    "productPage": [
+      "https://www.thomann.de/stairville_xbrick_quad_16x8w_rgbw.htm"
+    ],
+    "manual": [
+      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/c_385641_v4_en_online.pdf",
+      "https://images.thomann.de/pics/atg/atgdata/document/manual/c_385641_r4_en_online.pdf"
+    ],
+    "video": [
+      "https://video2.thomann.de/vidiot/02591c1c/video_i5704p10_yd59vqpa.mp4"
+    ]
+  },
+  "physical": {
+    "dimensions": [315, 88, 195],
+    "weight": 4.3,
+    "power": 125,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [25, 25]
+    }
+  },
+  "matrix": {
+    "pixelKeys": [
+      [
+        ["1A", "1B", null, "2A", "2B", null, "3A", "3B", null, "4A", "4B"],
+        ["1C", "1D", null, "2C", "2D", null, "3C", "3D", null, "4C", "4D"]
+      ]
+    ],
+    "pixelGroups": {
+      "All": "all",
+      "1": { "name": ["^1[A-D]$"] },
+      "2": { "name": ["^2[A-D]$"] },
+      "3": { "name": ["^3[A-D]$"] },
+      "4": { "name": ["^4[A-D]$"] }
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White $pixelKey": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    }
+  },
+  "availableChannels": {
+    "Intensity red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Intensity green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Intensity blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Intensity white": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity",
+        "brightnessStart": "0%",
+        "brightnessEnd": "100%"
+      }
+    },
+    "Stroboscope effect": {
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Strobe",
+        "helpWanted": "What are min/max strobe speeds?",
+        "speedStart": "0%",
+        "speedEnd": "100%"
```

I think the exact strobe frequencies are not that important, but we should indicate that with inexact keywords. However, it would be very useful to know when strobe is disabled.

```suggestion
        "helpWanted": "At which DMX values is strobe disabled?",
        "speedStart": "slow",
        "speedEnd": "fast"
```